ccd-definition/CaseEventToFields/CreateSDO-SDO-nonprod.json

```diff
@@ -113,12 +113,144 @@
   {
     "CaseTypeID": "CIVIL",
     "CaseEventID": "CREATE_SDO",
-    "CaseFieldID": "orderDetailsPlaceholder",
+    "CaseFieldID": "addNewDirections",
     "PageFieldDisplayOrder": 1,
-    "DisplayContext": "READONLY",
+    "DisplayContext": "OPTIONAL",
     "PageID": "OrderDetails",
     "PageDisplayOrder": 4,
```

Comment:
Could you change the `PageDisplayOrder` to `5` as this will be going on the disposal hearing page and similarly change the `PageFieldDisplayOrder` to `19` as it will be the last field on the page.